src/gbxml/Test/ReverseTranslator_GTest.cpp

```diff
@@ -968,3 +968,75 @@ TEST_F(gbXMLFixture, ReverseTranslator_Absorptance) {
     EXPECT_EQ(0.7, _material2->visibleAbsorptance());  // default
   }
 }
+
+TEST_F(gbXMLFixture, ReverseTranslator_v703gbXMLs) {
```

Comment:
Add a new test that does `gbXML -> model -> gbXML`. Then check that the latter gbXML is valid.